react/ActionMenu/index.jsx

```diff
@@ -56,8 +57,6 @@ const ActionMenuWrapper = ({
     >
       <ClickAwayListener onClickAway={onClose}>{children}</ClickAwayListener>
     </div>
-  ) : (
-    <BottomDrawer onClose={onClose}>{children}</BottomDrawer>
```

Comment:
Why this change ? All the hooks call after the early return are conditional now :/

Reply:
Oh you right, `inline` can change during life cycle. I'll go deeper and extract the react hooks in a new component to fix the point. 

The idea was : 
We don't change viewport size pretty often, and in mobile at least, we don't need to call `usePopper` and all.